docs/BUILD.md

```diff
@@ -5,6 +5,10 @@
 - [Release workflow](#release-workflow)
   - [Step 1: Pull request](#step-1-pull-request)
   - [Step 2: Release library in Github](#step-2-release-library-in-github)
+  - [Step 3: Publish to Maven central](#step-3-publish-to-maven-central)
```

Comment:
I wonder why we are not automating the release from "OSSRH" to "maven central" ? In that way, we don't need to document or do extra steps.
 
Check step 5: https://blog.sonatype.com/what-publishers-need-to-know-about-migrating-from-jcenter-/-bintray-to-the-central-repository

Reply:
217cbaa31